packages/FormElement/src/FormElement.js

```diff
@@ -66,117 +70,91 @@ function FormElement(props) {
     isDisabled,
     isInline,
     isLabelVisuallyHidden,
-    isReadOnly,
     label,
     size,
     ...moreProps
   } = props;
 
   const extractedChildren = extractChildren(children, [
-    "FormElement.Description",
-    "FormElement.Error",
-    "FormElement.Help",
-    "FormElement.Instructions",
+    subComponentDisplayNames.Description,
+    subComponentDisplayNames.Error,
+    subComponentDisplayNames.Help,
+    subComponentDisplayNames.Instructions,
+    subComponentDisplayNames.Content,
   ]);
   const ariaDescriptionId = React.useRef(uuidv4()).current;
   const ariaErrorId = React.useRef(uuidv4()).current;
   const ariaInstructionsId = React.useRef(uuidv4()).current;
-  const hasError = !!extractedChildren["FormElement.Error"] && !!extractedChildren["FormElement.Error"].props.children;
   const uniqueInputId = React.useRef(uuidv4()).current;
-  const inputId = isNil(id) || id === "" ? uniqueInputId : id;
+  const hasError =
+    !!extractedChildren[subComponentDisplayNames.Error] &&
+    !!extractedChildren[subComponentDisplayNames.Error].props.children;
+  const idForLabel = isNil(id) || id === "" ? uniqueInputId : id;
   const refLabel = React.useRef(null);
 
+  const getClonedElement = (displayName, extraProps = {}) => {
+    return React.cloneElement(extractedChildren[displayName], extraProps);
```

Comment:
Should there be a `if (!extractedChildren[displayName]) return null;` line here?

Reply:
Ok in the case of the `FormElement.Error` or `FormElement.Description` yes that would cause an error, although I won't do they check on that line, rather at the render function level..

Reply:
Sure the only two components that were not covered were in the footer. So this covers that now